plugin/action/mask/mask.go

```diff
@@ -273,6 +282,9 @@ func (p *Plugin) Do(event *pipeline.Event) pipeline.ActionResult {
 		stats.GetCounter(*p.config.MetricSubsystemName, timesActivated).Inc()
 		p.logger.Infof("mask appeared to event, output string: %s", event.Root.EncodeToString())
 	}
+	if p.config.MaskedEventExtraField != "" && maskApplied {
```

Comment:
Let's move this before logging (281) to see result in logs

Reply:
It will help to avoid inconvenience.